### ISSUE TITLE
[Feature]: Language selection needs to be persisted

### ISSUE THREAD

**Issue by birkskyum** [solidjs/solid-docs#636]
### 📋 Issue

The current implementation of the language selector just inject the locale in the url, but as soon as one navigates to a page that isn't translated, thus viewing the english content, everything is forgotten about the previous language selection.

### 📋 General description or bullet points

It's necessary to persist the language selection in LocalStorage / Session / Cookie, so that it's not forgotten all the time.

### 🖥️ Reproduction of code samples in StackBlitz

This can be seen in the language selector implementation, or by enabling the pt-br dictionary and navigating to a page that isn't translated.

**Comment by danieljcafonso** [comment]
Yeah, this is indeed intentional in the MVP implementation right now. In the future, we plan to persist with the language selection, probably in some storage, so it sticks. But considering right now no language really exists, it was deemed okay to revert back to English

**Comment by LadyBluenotes** [comment]
Closing this as i18n will be handled by @jer3m01 with SolidBase
